[BUG]: App crashes on pressing Edit from previewer in large deck

### Checked for duplicates?

- [X] This issue is not a duplicate

### What are the steps to reproduce this bug?

- Have a large deck
- Open it in the Browser and select Preview
- Click the edit icon on the top right.

### Expected behaviour

The edit note window opens.

### Actual behaviour

The app crashes.

### Debug info

```text
AnkiDroid Version = 2.17beta4 (b6ed71fdc2bfceeed3d96e8d249f9fa53aefed12)

Backend Version = 0.1.34-anki23.12.1 (23.12.1 1a1d4d5419c6b57ef3baf99c9d2d9cf85d36ae0a)

Android Version = 13 (SDK 33)

ProductFlavor = full

Manufacturer = motorola

Model = moto g62 5G

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 13; moto g62 5G Build/T1SSIS33.1-75-7-4; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/121.0.6167.164 Mobile Safari/537.36

ACRA UUID = d808c2e4-92ce-40db-9491-1fdcb75b1f50

Crash Reports Enabled = true
```


### (Optional) Anything else you want to share?

Might be related to https://github.com/ankidroid/Anki-Android/issues/15380

The issue is not present when using the new previewer.

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [X] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

https://ankidroid.org/acra/app/1/bug/129091/report/362c9f3b-1684-4599-8082-acc6ea3f081d

```
java.lang.RuntimeException: android.os.TransactionTooLargeException: data parcel size 1321264 bytes
	at android.app.ActivityClient.activityStopped(ActivityClient.java:88)
	at android.app.servertransaction.PendingTransactionActions$StopInfo.run(PendingTransactionActions.java:143)
	at android.os.Handler.handleCallback(Handler.java:984)
	at android.os.Handler.dispatchMessage(Handler.java:104)
	at android.os.Looper.loopOnce(Looper.java:238)
	at android.os.Looper.loop(Looper.java:357)
	at android.app.ActivityThread.main(ActivityThread.java:8088)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:957)
Caused by: android.os.TransactionTooLargeException: data parcel size 1321264 bytes
	at android.os.BinderProxy.transactNative(Native Method)
	at android.os.BinderProxy.transact(BinderProxy.java:584)
	at android.app.IActivityClientController$Stub$Proxy.activityStopped(IActivityClientController.java:1303)
	at android.app.ActivityClient.activityStopped(ActivityClient.java:85)
	... 9 more
```

--- a/AnkiDroid/src/main/java/com/ichi2/anki/Previewer.kt
+++ b/AnkiDroid/src/main/java/com/ichi2/anki/Previewer.kt
@@ -198,7 +198,6 @@ class Previewer : AbstractFlashcardViewer() {
     }
 
     override fun onSaveInstanceState(outState: Bundle) {
-        outState.putLongArray("cardList", cardList.toLongArray())
         outState.putInt("index", index)
         outState.putBoolean("showingAnswer", showingAnswer)
         outState.putBoolean("reloadRequired", reloadRequired)

